test_ir.py

```diff
@@ -37,5 +37,68 @@ class DisplaySystem(jkt.KernelTests):
     {'code': '1:3', 'mime': {'text/plain':'[1] 1 2 3'}},
     ]
 
+class AbstractIRKernel(jkt.KernelTests):
```

Comment:
Ok, I reordered... somehow the kernel tester looks like it would be a good fit for pytest fixtures :-)